colibre/registration.py

```diff
@@ -118,3 +118,44 @@
     smhm.name = name
 
     setattr(self, f"stellar_mass_to_halo_mass_{aperture_size}_kpc", smhm)
+
+# iterate through available dust types
+dust_idx = 0
+total_dust_fraction = metal_mass_fraction_gas*0.
```

Comment:
Thanks for the pointers, quick question: would you prefer to have `total_dust_mass` undefined in the no explicit dust case, or to define it as a zero array so that dust scripts still run without crashing?

Reply:
Let's have it defined as a zero array but change the name to state that it wasn't found.